MainApp/views.py

```diff
@@ -145,7 +145,7 @@ def addFollower(self,request):
         return HttpResponseRedirect(reverse('MainApp:profile', args=[request.POST['follow']]))       
 
 
-    def removeFollower(self,request):
+    def removeFollower(self,request,default_reverse='MainApp:profile',arg=None):
```

Comment:
Added some default arguments to allow flexibility with where we redirect the user once we unfollow.

For instance if we unfollow a user from our "following" tab we do not want to be redirected back to the homepage. We would rather stay on the same page...so we can override the default redirect and redirect back to the same "followingtab" instead